README.rst

```diff
@@ -543,21 +502,51 @@ Just be sure to pass in this thresholds parameter wherever it applies. For examp
    fraction_noise_in_unlabeled_class = inv_noise_matrix[pu_class][1 - pu_class] 
 
 
-Now that you have `indices_of_label_errors`, you can remove those label errors and train on clean data (or only remove some of the label errors and iteratively use confident learning / cleanlab to improve results)
+Now that you have `indices_of_label_errors`, you can remove those label errors and train on clean data (or only remove some of the label errors and iteratively use confident learning / cleanlab to improve results).
 
 
-The Polyplex
-------------
+Citation and Related Publications
```

Comment:
So citation and related publictions is here, but at the bottom. got it. this works (ignore comment above)!

Reply:
Perhaps we could still make the wording a bit friendly and less academic, like

>  hey Cleanlab isn't just a github, its based on pee-reviewed research from top journals. if you are using this for research, here are the relevant Cleanlab papers to cite:

Reply:
added:

`cleanlab` isn't just a github, it's based on peer-reviewed research. Here are the relevant papers to cite: